libs/shared/registration/src/lib/shared-registration.tsx

```diff
@@ -0,0 +1,163 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import {
+  Box,
+  Grid,
+  TextField,
+  Typography,
+  InputAdornment,
+  Button,
+} from '@material-ui/core';
+import Person from '@material-ui/icons/Person';
+import LockIcon from '@material-ui/icons/Lock';
+import PhoneIcon from '@material-ui/icons/Phone';
```

Comment:
Import all icons in one import: 

import {
  Person,
  Lock as LockIcon,
  Phone as PhoneIcon,
} from '@material-ui/icons';